.cargo/config.toml

```diff
@@ -1,3 +1,4 @@
 [env]
-CC_wasm32_wasi = "/opt/wasi-sdk/bin/clang"
-AR_wasm32_wasi = "/opt/wasi-sdk/bin/ar"
+WASI_SDK = { value = ".cargo/wasi-sdk-19.0", relative = true }
```

Comment:
ℹ️ This was the fix needed for the rust update.
This environment variable helps the linker find the proper `sysroot` when handling C dependencies. Not entirely sure why it worked before.

Reply:
Ok I guess this isn't enough, I'll actually have to set the cflags here. Should be good after that though

Reply:
Ok there was also an issue with the allocator function we expose where it did not properly handle mem alignment